Apps/Contoso.Forms.Demo/Contoso.Forms.Demo/ModulePages/CrashesContentPage.xaml.cs

```diff
@@ -52,27 +52,22 @@ protected override async void OnAppearing()
             if (Application.Current.Properties.TryGetValue(TextAttachmentKey, out var textAttachment) &&
```

Comment:
Was that tested on Xamarin? This is shared code.

Reply:
Tested with Xamarin.Forms Puppet iOS and Demo Android.